src/main/java/marquez/api/Ownership.java

```diff
@@ -18,10 +18,10 @@
   @JsonCreator
   public Ownership(
```

Comment:
Remove ownership if possible. Check if we still use it, but we don't even use it through the API.